plugins/foreman_virt_who_configure/0.1/index.md

```diff
@@ -0,0 +1,173 @@
+
+# 2. Installation
+
+This plugins requires [Katello](https://theforeman.org/plugins/katello/index.html) to be installed in your Foreman instance. If you have it, simply install a package with the plugin (rpm only), run migrations and seed by running
+
+    yum install rubygem-foreman_virt_who_configure
+    foreman-rake db:migrate
+    foreman-rake db:seed
+
+Plugin package lives in [Katello repositories](https://fedorapeople.org/groups/katello/releases/yum/).
+
+If installed foreman from git, you can simply add foreman-virt_who_configure gem as one of the dependencies. Don't forget to run migrations and seed afterwards and obviously restart the Foreman. You still need to have Katello installed.
+
+Note for Satellite 6 users, the plugin is installed out of the box and no extra action is required.  
+
+# 3. Configuration
+
+There's nothing to configure for this plugin yet. See the next chapter.
+
+# 4. Usage (features description)
+
+The plugin adds a new link "Virt-who configurations" under Infrastructure menu. This page lists all virt-who configurations that were created in Foreman. To create a new configuration, click Create Config button. Once provide all information, the configuration is registered and you can deploy it. Each configuration provides a configuration script that you find after clicking on its name. The configuration script should be run on host where you want to deploy virt-who instance at, e.g. the same host where Foreman runs. Existing virt-who configurations can be deleted when no longer required. Plugin also provides monitoring widget for dashboard. Please see chapters below for more details on every feature.
+ 
+## 4.1 Creating
+
+In order to create new virt-who configuration, navigate to Insrastructure -> Virt-who configurations and click Create Config button. This takes you to form which guides you through what we need to know in order to successfully configure virt-who. Most of fields provide some inline help, please click the help icon for more information.
+
+Note that every virt-who configuration must exist in an organization. If you haven't selected any organization, you'll see Owner field in the form where you need to choose an Organization. If you already are in some organization context, the Owner field is not displayed and the virt-who configuration will be automatically created in this Organization scope.
+  
+Default values which are prefilled are recommended values which are usually not needed to be changed. This applies to especially to Interval, Hypervisor ID and Satellite server FQDN. The Satellite server FQDN is where the virt-who instance will send reports to so make sure this FQDN is reachable from the host, where the virt-who instance will run. The default value is determined based on what Setting foreman_url is configured to.
```

Comment:
it's actually Satellite, virt-who uses Satellite 5 and Satellite 6 terms in everywhere including man pages, so I kept this configuration attribute with "Satellite in it's name too", even though it can be Foreman with Katello URL as well.